syncthing_gtk/deviceeditor.py

```diff
@@ -93,7 +93,7 @@ def display_value(self, key, w):
 					if n["deviceID"] == self.id:
 						rids.append(r["id"])
 			# Create CheckButtons
-			for folder in reversed(sorted(self.app.folders.values(), key=lambda x : x["id"])):
+			for folder in reversed(sorted(list(self.app.folders.values()), key=lambda x : x["id"])):
```

Comment:
In hindsight, I'm pretty sure ```sorted()``` returns an entirely new list instead of sorting the list object in-place, so I'll test it first and remove the ```list()``` if it works.

Reply:
edit: correct.  this can be removed